hack/deploy/releases/csi-linode-v0.0.1.yaml

```diff
@@ -109,27 +109,40 @@ spec:
           env:
             - name: ADDRESS
               value: /var/lib/csi/sockets/pluginproxy/csi.sock
-          imagePullPolicy: "IfNotPresent"
+          imagePullPolicy: "Always"
           volumeMounts:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
         - name: linode-csi-plugin
           image: displague/linode-csi-plugin:v0.1.1
           args :
+            - init
+            - "--region=$(LINODE_REGION)"
             - "--endpoint=$(CSI_ENDPOINT)"
             - "--token=$(LINODE_ACCESS_TOKEN)"
             - "--url=$(LINODE_API_URL)"
+            - "--node=$(NODE_NAME)"
+            - "--v=10"
           env:
             - name: CSI_ENDPOINT
               value: unix:///var/lib/csi/sockets/pluginproxy/csi.sock
             - name: LINODE_API_URL
               value: https://api.linode.com/
+            - name: NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
```

Comment:
@displague This is very useful. I should be able to use this on the cloud controller side too to avoid using the hostname hack